solutions/core-landing-zone/lz-folder/services-infrastructure/dns-project/dns.yaml

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 #########
 # Public Core DNS zone
-# SC-22
+# SC-20 - This is Parent zone and purely related to core landing zone only. Supports DNSSEC from spoofing attacks
+# Client applications will be using sub zones created under parent zone, however, their configuration will be application specific and will require seperate assessment
```

Comment:
Should this resource still be included if the intent is to deploy it manually to ensure the correct shard is assigned?

Reply:
the issue you mentionned is specific to the SSC landing zone. I suggest we don't mention it in this public package